src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/__init__.py

```diff
@@ -4,19 +4,28 @@
 # --------------------------------------------------------------------------------------------
 
 import azure.cli.command_modules.batch._help  # pylint: disable=unused-import
+import azure.cli.core.azlogging as azlogging
+
+
+__version__ = "3.1.0"
```

Comment:
Was this there initially? If so I missed it...
It won't get updated when we release and also is it used anywhere?